docs/macos/readme.md

```diff
@@ -0,0 +1,155 @@
+# Check for prerequisites
+All requisites should be available for your distribution. The most important are:
+
+* [Git] (https://git-scm.com/downloads)
+* [Docker] (https://docs.docker.com/engine/installation/)
+* [Docker Compose] (https://docs.docker.com/compose/install/)
+
+# Installation
+- Clone and download the project to your working directory:
+
+```
+$ git clone https://github.com/npbtrac/enpii-dev-suite
+```
+
+- Go to the working directory:
+
+```
+$ cd enpii-dev-suite
+```
+
+The current working folder will be `enpii-dev-suite`. After all the prerequisites have been installed, run the following commands to start the installation of docker images.
+  - This command will check if there’s .env file, if there isn’t, it will init and generate an .env file. Then you can repair values that need to be updated located on your based folder.
+
+```
+$ sh scripts/init-env-file.sh
+```
+
+  - When environment variables have been updated, we will generate the docker-compose.yml and configuration files (nginx conf - How to modify nginx conf can be found here #).
+
+```
+$ sh scripts/init-dev-suite.sh
+```
+
+  - Then download and bring up all the containers by the following command:
+
+```
+$ docker-compose up -d --build
+```
+
+- Wait for several mins and you'll have:
+  - Nginx (work as a webserver)
+  - PHP-FPM (PHP execution server)
+  - MySql
+  - Phpmyadmin
+- When it finishes downloading, we can check the list of all services by the command:
+
+```
+$ docker-compose ps
+```
+
+# Configure the nginx conf
+- Firstly, create two files called `index.html` and `index.php` within folder `<path-to-your-root-directorty>` and input the content:
```

Comment:
It should be `<path-to-document-root-directory>` not `<path-to-your-root-directorty>`.
You should guide content for index.html and info.php (should not be index.php)
- index.html
```
<h1>Webserver works</h1>
```
- info.php
```
<?php echo phpinfo();
```